deps/verifier/Cargo.toml

```
@@ -80,6 +80,8 @@ ccatoken = { git = "https://github.com/veraison/rust-ccatoken", rev = "6d5b8db",
 ear = { version = "0.4.0", optional = true }
 x509-parser = { version = "0.18.0", optional = true }
 reqwest.workspace = true
+reqwest-middleware = "0.4.2"
+http-cache-reqwest = { version = "0.16.0", features = ["manager-moka"] }
```

maybe disable `cacache:
```suggestion
http-cache-reqwest = { version = "0.16.0", default-features = false, features = ["manager-moka"] }
```